syntax = "proto3";
package dfi;

message DFICreateRingOnBufferResponse {
	message Segment {
        uint64 offset = 1;
        uint64 size = 2;
    }
    string name = 1;
    uint64 node_id = 2;
    Segment segment = 3;
    uint64 segmentsPerWriter = 4;
    uint64 segmentSizes = 5;
    uint32 return = 6;
}